### ISSUE TITLE
"File not found" error when no torrents are in the list

### ISSUE THREAD

**Issue by brainx64** [kzahel/jstorrent-legacy-app#155]
![2017-02-21_1237](https://cloud.githubusercontent.com/assets/3398372/23161478/f3319e6a-f832-11e6-9980-62d4b93d4317.png)

**Comment by kzahel** [comment]
Did you delete a torrent shortly beforehand?

**Comment by brainx64** [comment]
Yep, it popped up right after I had removed the last torrent. Also, the same notification always appears after I add any torrent.

**Comment by brainx64** [comment]
This short video captured from my laptop will better explain what I'm talking about - https://www.screencast.com/t/30Yj6mRlxYT

**Comment by brainx64** [comment]
It also happens when something finished downloading:

![2017-02-22_1149](https://cloud.githubusercontent.com/assets/3398372/23206090/026cc446-f8f5-11e6-9871-23af5bbc707c.png)

**Comment by kzahel** [comment]
The legacy Chrome App is no longer maintained. I've rebuilt JSTorrent from scratch — the new version is ready at **[jstorrent.com](https://jstorrent.com)**

If you run into any issues, please leave feedback!
